src/routes/+error.svelte

```diff
@@ -0,0 +1,7 @@
+<script lang="ts">
+  import { page } from '$app/stores';
+</script>
+
+<div class="flex h-screen justify-center items-center">
+  <h1 class="text-2xl font-semibold">{$page.status} | {$page.error?.message}</h1>
+</div>
```

Comment:
Add a newline please. Maybe turn on auto format in VSCode? Make sure you have prettier and the svelte extensions installed